python/lsst/pipe/base/argumentParser.py

```diff
@@ -564,8 +564,10 @@ def __init__(self, pattern):
 
                     def write(self, str):
                         str = str.rstrip()
-                        if str and fnmatch.fnmatch(str, self._pattern):
-                            print str
+                        # Strip off doc string line(s) and cut off at "=" for string matching
+                        matchStr = str.split("\n")[-1].split("=")[0]
```

Comment:
But I don't think that will leave `str` modified (i.e rstrip-ed), and I believe that is still desirable for the printing of `str`, so I think I will leave this one as is.